gprofiler/profilers/java.py

```diff
@@ -121,6 +123,58 @@ def get_ap_log(self) -> str:
         return self._ap_log
 
 
+JAVA_VERSION_TIMEOUT = 5
```

Comment:
PEP8's convention of prefixing private variables/function applies to globals as well

Reply:
Yup